pkg/detectors/sentrytoken/sentrytoken_test.go

```diff
@@ -174,6 +209,9 @@ const responseBody403 = `
   }
 ]
 `
+	reponseAccountDeactivated = `{"datail": "Authentication credentials were not provided"}`
```

Comment:
should this be `detail`?

Reply:
ah yea, it shouldn't affect the test since we are expecting a completely different structure. Nonetheless, fixed. TY.